internal/twirptest/service.twirp.go

```diff
@@ -565,33 +540,68 @@ func twirpErrorFromIntermediary(status int, msg string, bodyOrLocation string) t
 	}
 	return twerr
 }
+
 func isHTTPRedirect(status int) bool {
 	return status >= 300 && status <= 399
 }
 
-// wrappedError implements the github.com/pkg/errors.Causer interface, allowing errors to be
-// examined for their root cause.
+// wrapInternal wraps an error with a prefix as an Internal error.
+// The original error cause is accessible by github.com/pkg/errors.Cause.
+func wrapInternal(err error, prefix string) twirp.Error {
+	return twirp.InternalErrorWith(&wrappedError{prefix: prefix, cause: err})
+}
+
 type wrappedError struct {
-	msg   string
-	cause error
+	prefix string
+	cause  error
+}
+
+func (e *wrappedError) Cause() error  { return e.cause }
+func (e *wrappedError) Error() string { return e.prefix + ": " + e.cause.Error() }
+
+// handlePanics makes sure that panics during rpc methods respond with Twirp Interna
+// error responses (status 500) and trigger hooks with the panic wrapped as an error.
+// The panic is not intercepted and should be handled in middleware to avoid empty responses.
+func handlePanics(ctx context.Context, resp http.ResponseWriter, hooks *twirp.ServerHooks) {
+	if r := recover(); r != nil {
+		err := errFromPanic(r)                                                 // allow to inspect error on Error hooks
```

Comment:
These comments are overflowing way off to the right side. Could we get them to stay under ~100 characters, either by moving them above the lines they describe or by not vertically aligning them?

Reply:
They seem to auto-align when generating code. I didn't realize that was happening. Fixing ...